scripts/tagcloud.js

```diff
@@ -20,12 +20,13 @@ function get_tags (url) {
   var not_display3 = not_display4 + ' extension'
   var dontarray = ['view page', 'open', 'read more', not_display1, not_display2, not_display3, not_display4]
   var new_url = 'https://archive.org/services/context/tagcloud?url=' + toBeUsedAsURL
-  $('#loader_tagcloud').show()
+  $('#container-wordcloud').hide()
   fetch(new_url)
     .then(response => response.json())
     .then(function (data) {
+      $('#container-wordcloud').show()
```

Comment:
the .show() on line 27 should be after the if on line 29.

Reply:
YES, correct. I am changing it sorry !1